simcase-game/src/components/LastScenario.js

```diff
@@ -1,12 +1,24 @@
+/** @jsx jsx */
 import React from "react";
+import { jsx } from "@emotion/core";
+
+import { Button } from "../components/ui";
+
+const button = {
+  margin: "5px 0"
+};
 
 function LastScenario({ confirmSubmit, closeSubmit }) {
   return (
-    <div>
+    <>
       <p>You are going to submit your answer. Are you sure?</p>
-      <button onClick={confirmSubmit}>Finish and submit</button>
-      <button onClick={closeSubmit}>No, I want to continue</button>
-    </div>
+      <Button css={button} onClick={confirmSubmit}>
+        Finish and submit
+      </Button>
+      <Button css={button} onClick={closeSubmit}>
+        No, I want to continue
```

Comment:
I think this is not the message for this button. If the users press `No`, then their email address is not required and the leaderboard with the top 5 players is shown.